src/main.js

```diff
@@ -1 +1,24 @@
-console.log("Car");
+import * as io from "./view/inputoutput.js"
+import Race from "./race.js";
+import Car from "./car.js"
+
+const main = async () => {
+    const readline = io.createInterface();
+
+    const names = await io.receivedCarNames(readline);
+
+    const cars = [];
+    for (const name of names) {
+        cars.push(new Car(name));
+    }
```

Comment:
map을 통해 더 선언적으로 나타낼 수 있을 것 같습니다.


```suggestion
    const cars = names.map(name => new Car(name))
```